doc/bootstrapping.txt

```diff
@@ -25,10 +25,10 @@ bootstrap.Bootstrap starts with sanity checks:
  * that there is at least one authorised SSH key
  * that there is a CA Cert and CA Key
  * that the model storage is writable (by writing the bootstrap-init file)
- * finds available tools
-   - locate tools available externally (matching constraints)
-   - determine which tools can be built and uploaded to make up shortfall in above
-   - if the best tools are made locally, and we can upload tools, they get uploaded
+ * finds available agent binaries
+   - locate agent binaries available externally (matching constraints)
+   - determine which agent binaries can be built and uploaded to make up shortfall in above
+   - if the best agent binaries are made locally, and we can upload agent binaries, they get uploaded
```

Comment:
This reads a bit off. "if we can find them, and can upload them, we upload them".
- if the best agent binaries are local, we attempt to upload them
?